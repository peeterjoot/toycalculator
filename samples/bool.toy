BOOL i1v;
i1v = TRUE;
BOOL i2v;
i2v = FALSE;
PRINT i2v;
i2v = i1v;
FLOAT64 f;
f = i1v;
PRINT f;
PRINT i2v;
EXIT i1v;
