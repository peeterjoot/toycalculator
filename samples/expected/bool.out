0
1.000000
1
